testgrid/config.yaml

```diff
@@ -4594,56 +4594,76 @@ dashboards:
     test_group_name: ci-kubernetes-bazel-test
+    description: TODO: think this is owned by sig-testing, set an alert up; TODO: what file does this come from; TODO: what does this do
   - name: verify-master
     test_group_name: ci-kubernetes-verify-master
+    description: TODO: think this is owned by sig-testing, set an alert up; TODO: what file does this come from; TODO: what does this do
   - name: node-kubelet-master
     test_group_name: ci-kubernetes-node-kubelet
+    description: TODO: think this is owned by sig-node, set an alert up; TODO: what file does this come from; TODO: what does this do
   - name: gce-cos-master-default
     test_group_name: ci-kubernetes-e2e-gci-gce
+    description: TODO: think this is owned by sig-release release-team subproject, set an alert up; TODO: what file does this come from; TODO: what does this do
   - name: kops-aws-master
     test_group_name: ci-kubernetes-e2e-kops-aws
+    description: TODO: think this is owned by sig-cluster-lifecycle kops subproject, set an alert up; TODO: what file does this come from; TODO: what does this do
   - name: Conformance - GCE
-    description: Runs conformance tests using kubetest against latest kubernetes master CI build on GCE
     test_group_name: ci-kubernetes-gce-conformance
+    description: TODO: unsure if this should be here; TODO: what file does this come from; runs conformance tests using kubetest against latest kubernetes master CI build on GCE
   - name: Conformance - OpenStack
-    description: Runs conformance tests using kubetest against latest kubernetes master with cloud-provider-openstack
     test_group_name: ci-periodic-cloud-provider-openstack-acceptance-test-e2e-conformance
+    description: TODO: unsure if this should be here; TODO: what file does this come from; runs conformance tests using kubetest against latest kubernetes master with cloud-provider-openstack
   - name: Conformance - vSphere-cloud-provider
-    description: Runs conformance tests using kubetest against latest kubernetes master with cloud-provider-vsphere
     test_group_name: ci-periodic-cloud-provider-vsphere-test-e2e-conformance
+    description: TODO: unsure if this should be here; TODO: what file does this come from; Runs conformance tests using kubetest against latest kubernetes master with cloud-provider-vsphere
   - name: Conformance - vSphere
-    description: Runs conformance tests using e2e.test against latest kubernetes master with in-tree vSphere
     test_group_name: ci-periodic-vsphere-test-e2e-conformance
+    description: TODO: unsure if this should be here; TODO: what file does this come from; Runs conformance tests using e2e.test against latest kubernetes master with in-tree vSphere
   - name: gce-device-plugin-gpu-master
     test_group_name: ci-kubernetes-e2e-gce-device-plugin-gpu
+    description: TODO: think this is owned by sig-gcp, sig-scheduling, or sig-node, set an alert up; TODO: what file does this come from; TODO: what does this do
```

Comment:
sig-gcp because that's the file that contains this job, but I feel like this is wg-resource job, and I'm not sure which sig is most appropriate in that context?

Reply:
@mindprince, I remember you helped set this up, thoughts?

SIG-Node maybe? At least in rel-notes that's where we're putting device plugin updates recently
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.13.md#sig-node

Aside: Do we want allow WG to own release tests?

Reply:
> Aside: Do we want allow WG to own release tests?
Oh good point. I would be a bit cautious doing this; the time bounded nature of working groups means that tests could risk losing their owners. At the same time, the earlier we start running e2e tests the better (in my opinion)... Perhaps something like "wg can own tests while alive, but are responsible for decomposing/hand over to appropriate sigs when disbanded"? 🤔

Reply:
The tests use `[Feature:GPUDevicePlugin]` for their regex, which lives in `test/e2e/scheduling/nvidia-gpus.go`